.changeset/tiny-toes-fail.md

```diff
@@ -0,0 +1,5 @@
+---
+'@rocket.chat/meteor': patch
+---
+
+Added new methods `findRoomsByAgentId` and `countRoomsByAgentId` to livechat bridge
```

Comment:
```suggestion
Introduced new methods for Rocket.Chat Apps to interact with livechat: `findRoomsByAgentId` and `countRoomsByAgentId`
```

Just a more "user friendly" message, mentioning more explicitly what feature is impacted